app/models/report.rb

```diff
@@ -20,6 +20,7 @@ class << self
 
   COLUMN_SELECTOR_COLUMN_SIZE = 3 # Helps display report columns in neatly lined up columns.
   ROWS_PER_PAGE = 100
```

Comment:
Because we are only getting back from Solr the specific fields needed for the report, this could be bumped up to 200 or 250 for even better performance for large reports

Reply:
If I understand correctly, this is the number of results when a user scrolls through the report in Argo. A larger number might make this laggier since a larger response is going to be slower (though they could scroll through more results before a new page is requested). Since there hasn't been a request to change the report in Argo and I don't know why this number was selected, I'm not inclined to make a change at this time.

Reply:
@mjgiarlo looks like this is your code. Feel free to chime in if you feel this is worth further consideration.

Reply:
You re correct, @justinlittman. Are your changes deployed anywhere? I'd be happy to do a quick test of the UI view to ensure the 10K change doesn't affect it, unless you've already done this.

Reply:
I just checked (in prod). The Report UI only pulls back 100 records.

Reply:
Super, thank you!